src/FFTW.jl

```diff
@@ -2,7 +2,7 @@ __precompile__()
 
 module FFTW
 
-const depsfile = joinpath(dirname(@__FILE__), "..", "deps", "deps.jl")
+const depsfile = joinpath(dirname(@__DIR__), "deps", "deps.jl")
```

Comment:
I don't really understand why we do this rather than `joinpath("..", "deps", "deps.jl")`, since `include` is always relative to the path of the current file.